Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -175,6 +175,31 @@ + (BOOL)cameraIsPresent {
     return [AVCaptureDevice defaultDeviceWithMediaType:AVMediaTypeVideo] != nil;
 }
 
++ (BOOL)hasCamera:(MTBCamera)camera {
+    AVCaptureDevicePosition position = [self devicePositionForCamera:camera];
+
+    for (AVCaptureDevice *device in [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo]) {
+        if (device.position == position) {
+            return YES;
+        }
+    }
+
+    return NO;
+}
+
++ (MTBCamera)oppositeCameraOf:(MTBCamera)camera {
```

Comment:
In that case, `flipCamera` is also something that must be reconsidered... we should give this a try on a 7 Plus, but I don't expect issues.